wordpress-mobile/gutenberg-mobile
# Use Hermes bytecode on Android

Fixes #1660

This PR is a continuation of https://github.com/wordpress-mobile/gutenberg-mobile/pull/2003, with a simpler approach and just using the bytecode version directly. If it proves more successful I will close the other PR.

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/26732
WPAndroid PR: https://github.com/wordpress-mobile/WordPress-Android/pull/13297

I've left a comment with the (green!) [results of my Writing Flow manual tests here](https://github.com/WordPress/gutenberg/pull/26732#issuecomment-723043019).

#### Changes in this PR

* Use the bytecode version of the JS bundle for the apps and the tests

#### To test:
* No special testing other than all operations should look and feel faster.
* A perhaps easier to spot improvement is the editor startup time. Use the WPAndroid PR/build to start a new blog post and notice how much faster the editor comes online.

PR submission checklist:
